addon/plugins/table/nodes/table.ts

```diff
@@ -106,24 +110,78 @@ interface TableNodes extends Record<string, NodeSpec> {
   table_header: NodeSpec;
 }
 
+const appendToStyleAttribute = (
+  attributes: Record<string, unknown>,
+  value: string,
+) => {
+  if (!attributes.style) {
+    attributes.style = value;
+    return;
+  }
+
+  if (typeof attributes.style === 'string') {
+    attributes.style = `${attributes.style}; ${value};`;
+    return;
+  }
+
+  return;
+};
+
+const getDefaultCellAttributes = ({
+  inlineBorderStyle,
+}: {
+  inlineBorderStyle?: string;
+} = {}): Record<string, ExtraAttribute> => ({
+  background: {
+    default: null,
+    getFromDOM(dom) {
+      return dom.style.backgroundColor || null;
+    },
+    setDOMAttr(value, attrs) {
+      if (typeof value === 'string') {
+        appendToStyleAttribute(attrs, `background-color: ${value}`);
+      }
+    },
+  },
+  borderLeft: {
+    default: inlineBorderStyle ?? null,
+    getFromDOM(dom) {
+      return dom.style.borderLeft || null;
+    },
+    setDOMAttr(value, attrs) {
+      if (typeof value === 'string') {
+        appendToStyleAttribute(attrs, `border-left: ${value}`);
+      }
+    },
```

Comment:
Nice approach to include seperate `fromDOM` and `toDOM` methods for each of the attributes. I was already thinking of including that option in the default `SayNodeSpec` API.